### DISCUSSION TITLE
How to connect Azure SQL database from Slick?

### DISCUSSION THREAD

**Discussion by Ashutosh1198** [slick/slick#2438] [Questions]
How to connect Azure SQL database from Slick application?

I was  using jTDS 1.3.1 to connect **MS SQL Server** database hosted different server.
As client moving to azure managed instance (azure cloud) , database moved to azure SQL database.

But application generating Error:
ERROR: "Login failed due to client TLS version being less than minimal TLS version allowed by the server" when connecting to Azure SQL Database"

Is there any way to connect azure SQL database from Slick ?

**Comment by nafg** [comment]
Sounds like a JDK version or driver issue rather than slick, can you get JDBC to work at all?
